Sources/OpenAPIRuntime/Base/CopyOnWriteBox.swift

```diff
@@ -26,15 +26,15 @@
 
         /// Creates a new storage with the provided initial value.
         /// - Parameter value: The initial value to store in the box.
-        @inlinable init(value: Wrapped) { self.value = value }
+        @usableFromInline init(value: Wrapped) { self.value = value }
     }
 
     /// The internal storage of the box.
     @usableFromInline internal var storage: Storage
 
     /// Creates a new box.
     /// - Parameter value: The value to store in the box.
-    @inlinable public init(value: Wrapped) { self.storage = .init(value: value) }
+    public init(value: Wrapped) { self.storage = .init(value: value) }
```

Comment:
Discovered by building the runtime library with library evolution enabled - you can't inline the designated initializer and still be resilient. The only alternative is to mark the struct as frozen.